gosnippets/UltiSnips/go.snippets

```diff
@@ -159,6 +159,19 @@ if err != nil {
 ${0}
 endsnippet
 
+# formatted error snippet
+snippet errf "Formatted error return " !b
+if err != nil {
+	return fmt.Errorf("${1:description}%v", err)
```

Comment:
Can you remove the `description texts from here ? Also from the other recurrences as well.